build.sbt

```diff
@@ -874,6 +883,108 @@ lazy val zioServer: ProjectMatrix = (projectMatrix in file("server/zio-http4s-se
   .jvmPlatform(scalaVersions = allScalaVersions)
   .dependsOn(zio, http4sServer, serverTests % Test)
 
+// serverless
+
+lazy val awsLambda: ProjectMatrix = (projectMatrix in file("serverless/aws/lambda"))
+  .settings(commonJvmSettings)
+  .settings(
+    name := "tapir-aws-lambda",
+    libraryDependencies ++= loggerDependencies,
+    libraryDependencies ++= Seq(
+      "com.softwaremill.sttp.client3" %% "http4s-ce2-backend" % Versions.sttp,
+      "org.http4s" %% "http4s-blaze-client" % Versions.http4s
```

Comment:
probably a follow-up task: I think we are always using this with Java 11+? If so, we might be able to use the `HttpClient` backend, which would reduce the dependencies of this project (to just cats-effect and regular java)

Reply:
I wrote it down